src/main/kotlin/frc/robot/scoreState/AmpState.kt

```diff
@@ -38,15 +40,17 @@ class AmpState : ScoreState {
     private fun end(): Runnable {
         return Runnable {
             gripper.feed()
-                .andThen(shooter.stop())
-                .alongWith(conveyor.stop())
+                .andThen(
+                    shooter.stop()
+                        .alongWith(conveyor.stop())
+                )
```

Comment:
This is a nice use case for [infix functions](https://kotlinlang.org/docs/functions.html#infix-notation):
```suggestion
            (gripper.feed() andThen shooter.stop()) alongWith conveyor.stop()
```
I have also added parentheses for clarity.
I copied [Hamosad's extensions module](https://github.com/Hamosad1657/HamosadLib/blob/cddb9f7d4e82d2a16639816b74b732ca1484538c/src/main/kotlin/com/hamosad1657/lib/commands/Extentions.kt).